### PR TITLE
updated sample snippets in MD files so they are in sync with template-examples repo

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

File: docs/toc.yml [hunk 1]
```diff
@@ -1607,8 +1607,6 @@ items:
                     href: guide/azure-resource-manager/advanced-templates/index.md
                   - name: Update a resource
                     href: guide/azure-resource-manager/advanced-templates/update-resource.md
-                  - name: Conditionally deploy a resource
-                    href: guide/azure-resource-manager/advanced-templates/conditional-deploy.md
```
Comment (ckittel) [inline docs/toc.yml:1611]: We should be deleting the `conditional-deploy.md` file and adding a redirect as well as part of this PR.

**Review by ckittel — CHANGES_REQUESTED** [review]
Thanks for the freshness pass.  Here are a couple tweaks before we bring it into the repo. LMK if you have any questions

**Comment by PRMerger16** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

**Comment by PRMerger10** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

**Comment by PRMerger18** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

**Comment by PRMerger6** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

**Comment by ckittel** [comment]
I'll pull it into a PR on `-pr` and get a review link

**Comment by PRMerger20** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

**Comment by PRMerger14** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

**Comment by PRMerger13** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

**Comment by PRMerger19** [comment]
@magrande : Thanks for your contribution! The author(s) have been notified to review your proposed change. @PeterTaylor9999

**Comment by ckittel** [comment]
Taken internally to MicrosoftDocs/architecture-center-pr#3683, closing this PR.

**Pull request closed without merging**